src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,37 @@
+package tasks;
+
+import java.io.IOException;
+
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String toString() {
+        return getStatusIcon() + getDescription();
+    }
```

Comment:
Maybe this should be override

Reply:
Hi Ethan, Task is an abstract class so wouldn't the method of toString() within task be 'Override' by other classes extending on it?

Reply:
toString() is an inbuilt java object method so you are overriding it when you invoke this method (which you did in deadline class)

Reply:
Alright noted